Feliz.UseMediaQuery/UseMediaQuery.fs

```diff
@@ -0,0 +1,65 @@
+﻿module Feliz.UseMediaQuery
+
+open System
+open Browser
+open Browser.Types
+open Feliz
+
+type ScreenSize =
```

Comment:
Can you please add `[<RequireQualifiedAccess>]` to this union?

Reply:
Ok, fixed